flox-bash/lib/templateFloxEnv/pkgs/default/flox.nix

```diff
@@ -4,16 +4,31 @@
   # packages.nixpkgs-flox.figlet = {};
   # packages.nixpkgs-flox.bat = { version = "0.22.1"; };
 
+  # Activation Extensions
+  # Provides an extensible `<env>/etc/profile` script.
+  # packages.flox.etc-profiles = {
+  #   # All language packs are installed by default, but you can also
```

Comment:
what does "language pack" mean?

Reply:
Also when I add a language pack, I still have to add the respective tool chain right?

Reply:
It's a new concept, and yes as of right now it's incumbent upon the user to know that they need to install it. Alternatives would include adding logic to flox itself to automatically add language packs when installing packages that need it, publishing new packages comprised of interpreters + language packs bundled together, or automatically installing all language packs by default for environments of type "developer" (defined/identified by whatever means makes the most sense).

Reply:
@ysndr I pushed an update to `crates/flox/doc/flox-activate.md` that documents the `etc-profiles` feature in some detail. I'm sure we'll need more (and in other places) but that at least gets the process started. With that can we get this approved and merged? Thanks.